presto-orc/src/main/java/com/facebook/presto/orc/metadata/Encryption.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc.metadata;
+
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class Encryption
```

Comment:
You mentioned somewhere that this class will be different for ORC, hence, adding an interface and Dwrf implementation would be appropriate.

Reply:
The reason I decided against it is that there are a number of differences, and it's not clear what would go in a common interface.  I think it's the right decision long term, but I figure it should wait for the ORC implementation to figure out what that interface should look like (is it empty and we just cast to the appropriate encryption type for each, or are ther common methods?).  what if I rename this DwrfEncryption so that it's clear it's separate from the OrcEncryption?

Reply:
@rschlussel 

> what if I rename this DwrfEncryption so that it's clear it's separate from the OrcEncryption?

Good idea. Let's do that.